broker-daemon/broker-rpc/broker-rpc-server.js

```diff
@@ -79,21 +52,24 @@ class BrokerRPCServer {
     this.pubKeyPath = pubKeyPath
     this.privKeyPath = privKeyPath
     this.disableAuth = disableAuth
+    this.rpcUser = rpcUser
+    this.rpcPass = rpcPass
+    this.auth = basicAuth.verify.bind(this)
```

Comment:
why are you binding to the current context? Is there any reason that you need to?

Reply:
@treygriffith we bind to the current context because `verify` is called from within grpc-methods but is unable to access (or pass in arguments) for user/pass